src/commonMain/kotlin/fr/acinq/lightning/serialization/v1/ChannelState.kt

```diff
@@ -169,7 +169,7 @@ data class LocalParams constructor(
 ) {
     constructor(from: fr.acinq.lightning.channel.LocalParams) : this(
         from.nodeId,
-        from.fundingKeyPath,
+        from.channelKeys.fundingKeyPath,
```

Comment:
So we keep serializing with the old format and re-compute the channel keys when we load.
The main reason is to avoid storing private keys directly in the channels DB, right?
It may be worth a comment to ensure we don't "fix this" by mistake in the future.

Reply:
Exactly, I added an explanation.